src/lang/english.txt

```diff
@@ -2778,6 +2778,7 @@ STR_SAVELOAD_DETAIL_NOT_AVAILABLE                               :{BLACK}No infor
 STR_SAVELOAD_DETAIL_COMPANY_INDEX                               :{SILVER}{COMMA}: {WHITE}{STRING1}
 STR_SAVELOAD_DETAIL_GRFSTATUS                                   :{SILVER}NewGRF: {WHITE}{STRING}
 STR_SAVELOAD_FILTER_TITLE                                       :{BLACK}Filter string:
+STR_SAVELOAD_DIFFERENT_ID                                       :{RED}Not the same game as you are saving.
```

Comment:
this text feels awkward, though i can't come up with anything better off hand

Reply:
It is a really awkward situation to attempt to describe concisely. I'll try changing it to "Overwriting a different game file."